### PR TITLE
Gas optimization

### PR DESCRIPTION
**Pull request by Freitag-fri**

### CODE REVIEW

File: contracts/Peeranha.sol [hunk 1]
```diff
@@ -95,7 +95,7 @@ contract Peeranha is IPeeranha, Initializable {
      *
      * - Must be follow the community.  
      */
-    function unfollowCommunity(uint32 communityId) external onlyExisitingUser(msg.sender) override {
+    function unfollowCommunity(uint32 communityId) external override {
```
Comment (sergeyi23) [inline contracts/Peeranha.sol:98]: Why we do not check existing user here

File: contracts/Peeranha.sol [hunk 2]
```diff
@@ -83,7 +83,7 @@ contract Peeranha is IPeeranha, Initializable {
      *
      * - Must be an community.  
      */
-    function followCommunity(uint32 communityId) external onlyExisitingUser(msg.sender) override 
+    function followCommunity(uint32 communityId) external override
```
Comment (sergeyi23) [inline contracts/Peeranha.sol:86]: Why no check for existing user?

File: contracts/libraries/CommonLib.sol [hunk 3]
```diff
@@ -7,7 +7,7 @@ import "@openzeppelin/contracts-upgradeable/utils/SafeCastUpgradeable.sol";
 /// @notice
 /// @dev
 library CommonLib {
-  uint16 constant QUICK_REPLY_TIME_SECONDS = 900;
+  uint16 constant QUICK_REPLY_TIME_SECONDS = 6;
```
Comment (sergeyi23) [inline contracts/libraries/CommonLib.sol:10]: Why this changed

File: contracts/libraries/PostLib.sol [hunk 4]
```diff
@@ -71,6 +69,9 @@ library PostLib  {
         uint16 replyCount;
         uint16 deletedReplyCount;
         bool isDeleted;
+
+        uint8[] tags;
```
Comment (sergeyi23) [inline contracts/libraries/PostLib.sol:73]: Why move fields

File: contracts/libraries/UserLib.sol [hunk 5]
```diff
@@ -17,14 +17,14 @@ library UserLib {
 
   struct User {
     IpfsLib.IpfsHash ipfsDoc;
+    uint256 creationTime;
```
Comment (sergeyi23) [inline contracts/libraries/UserLib.sol:20]: Does moving this fields saves gas?

File: contracts/Peeranha.sol (hunk 1 above)
Comment (Freitag-fri) [reply contracts/Peeranha.sol:98]: we check him inside userLib

File: contracts/Peeranha.sol (hunk 2 above)
Comment (Freitag-fri) [reply contracts/Peeranha.sol:86]: we check him inside userLib

File: contracts/libraries/PostLib.sol (hunk 4 above)
Comment (Freitag-fri) [reply contracts/libraries/PostLib.sol:73]: save gas

File: contracts/libraries/UserLib.sol (hunk 5 above)
Comment (Freitag-fri) [reply contracts/libraries/UserLib.sol:20]: yes

**Pull request merged**